rust-src/bulletproofs/src/range_proof.rs

```diff
@@ -160,93 +123,108 @@ pub fn prove<C: Curve, T: Rng>(
     v_keys: &CommitmentKey<C>,
     randomness: &[Randomness<C>],
 ) -> Option<RangeProof<C>> {
+    // Part 1: Setup and generation of vector commitments
+    // V (for the values),
+    // A (their binary representation),
+    // S (the blinding factors)
     let nm = usize::from(n) * usize::from(m);
+    // TODO: Check that v_vec has the same length as randomness
+    // Check that we have enough generators for vector commitments
+    // TODO: Add proper error handling
```

Comment:
Remove the todo. There is nothing improper about the current error handling.

Reply:
What is meant here is that other functions return an error saying what went wrong, while this return none. It's in principle fine as is, but if we add the other checks, it seems to make sense to have different return values for different errors.

Reply:
The todo should not stay in the code, and there is nothing improper as is. Either resolve or remove.

TODOs left in code never get resolved in practice.

Reply:
Removed todos